master/buildbot/test/fake/docker.py

```diff
@@ -15,8 +15,12 @@
 
 
 class Client(object):
+    latest = None
 
     def __init__(self, base_url):
+        Client.latest = self
+        self.call_args_create_container = []
```

Comment:
that depends. It's not part of the original docker.Client API, but it is part of the public fake.Client API (it is used in the unit tests to access the arguments used when calling create_container & create_host_config). So, are we talking about the API of docker.Client or fake.Client?